spec/email_validator_spec.rb

```diff
@@ -764,6 +764,28 @@ class DefaultUserWithMessage < TestModel
       end
     end
 
+    context 'when `require_fqdn` is explicitly enabled' do
+      let(:opts) { { :require_fqdn => true } }
+
+      context 'when given a valid hostname-only email' do
+        let(:email) { 'someuser@somehost' }
+
+        context 'when in `:loose` mode (the default)' do
+          it 'is invalid using EmailValidator.valid?' do
+            expect(described_class).not_to be_valid(email, opts)
```

Comment:
This is wrong. `someuser@somehost` should always be expected to be valid in `loose` mode, because it has an `@` in it and that is all `loose` mode is looking for.